.goreleaser.yml

```diff
@@ -1,7 +1,4 @@
 project_name: minimock
-before:
-  hooks:
-    - make all
```

Comment:
Removed the hook to resolve the error.

- https://github.com/gojuno/minimock/pull/83#issuecomment-1876320069